tests/ext/extract_ip_addr.phpt

```diff
@@ -66,7 +66,8 @@ test('via', '1.0 127.0.0.1, HTTP/1.1 [2001::1]:8888');
 test('via', 'HTTP/1.1 [2001::1, HTTP/1.1 [2001::2]');
 test('via', '8.8.8.8');
 test('via', '8.8.8.8, 1.0 9.9.9.9:8888,');
-test('via', '1.0 bad_ip_address, 1.0 9.9.9.9:8888,');
+test('via', '1.0 pseudonym, 1.0 9.9.9.9:8888,');
+test('via', '1.0 172.32.255.1 comment');
```

Comment:
The RFC also specifies that after the IP/pseudonym there can be a comment. This comment was already skipped by the code but I did not find a test for it so I added it.